src/Minifiers/BladeMinifier.php

```diff
@@ -94,6 +97,8 @@ protected function containsBadHtml($value)
     {
         return preg_match('/<(code|pre|textarea)/', $value) ||
             preg_match('/<script[^\??>]*>[^<\/script>]/', $value) ||
+            preg_match('/skip\.minification/', $value) ||
+            preg_match('/Schema::create/', $value) ||
```

Comment:
Please don't add this. This is very specific, and there's no guarantee that that that alias name is even used.